docs/UsersGuide.asciidoc

```diff
@@ -580,6 +580,48 @@ http://openqa/tests/latest/badge
 There is an optional parameter 'show_build=1' that will
 prefix the status with the build number.
 
+=== Build a continuous openQA monitoring dashboard
+
+With just some common tools a dashboard showing the openQA index page with
+continuously updated results can be setup. Example setup using the
+`fullscreen=1` parameter:
```

Comment:
```suggestion
continuously updated results can be setup. Example setup using the
`fullscreen=1` parameter to remove navigation and branding:
```

Since afair this feature is not documented anywhere else, I suggest explaining what this does here.

Reply:
Try looking for the word **fullscreen** in the documentation 😉

How about

```suggestion
continuously updated results can be setup. Example setup using the
`fullscreen=1` parameter to use the entire page for the build results:
```